contracts/Mocks/OUSDMockToken.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+/// @title OUSD mock token
+///
+/// @dev This is the contract for the testing OUSD
+///
+/// TODO: add access control and roles
+///
+contract OUSDMockToken is ERC20("Mock OUSD", "oUSD"){
```

Comment:
What have you guys decided on syntax "oUSD" or "OUSD"?

Reply:
I'm in favor of "OUSD" because it's how the Origin Team formats the name of their token. It's also widely formatted that way on other sites. There are some unpopular coins that have the format oUSD and ioUSD that we don't want to confuse our users with. Screengrab is a search on coingeck for "OUSD".
<img width="392" alt="Screen Shot 2022-04-05 at 9 35 45 PM" src="https://user-images.githubusercontent.com/271327/161896524-923c38af-760e-49d6-8c1a-7f0cd5639791.png">